File: react/reactproject/src/models/schedule.ts
import { Turn } from "./turn";

export class Schedule {
    Turns: Turn[];
    ScheduleId: number|undefined;
    DoctorId: number;
    constructor(data: Partial<Schedule> = {}) {
        this.Turns = data.Turns ?? [];
        this.ScheduleId = data.ScheduleId;
        this.DoctorId = data.DoctorId ?? 0;
    }
}

File: Net/FinalProject/FinalProject.Clean/Models/Schedule.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FinalProject.Core.Models
{
    public class Schedule
    {
        [Key]
        public int ScheduleId { get; set; }
        public string DoctorName { get; set; }
        public List<Turn> turns { get; set; } = new List<Turn>();
    }
}
